### PR TITLE
feat(publick8s): add a staging fileshare for geoip-data

### PR DESCRIPTION
**Pull request by smerle33**
provide a staging storage share to test the cronjob for geoip update without impacting the production storage share (fileshare)

### CODE REVIEW

File: publick8s.tf [hunk 1]
```diff
@@ -367,6 +367,48 @@ module "cronjob_geoip_data_fileshare_serviceprincipal_writer" {
...
+  }
+
+  data = {
+    azurestorageaccountname = azurerm_storage_account.publick8s.name
+    azurestorageaccountkey  = azurerm_storage_account.publick8s.primary_access_key
+  }
+
+  type = "Opaque"
+}
+
```
Comment (dduportal) [inline publick8s.tf:398]:
```suggestion
```

=> uneeded as the secret is scoped to the storage account (e.g. it's the same for all file shares of a given storage account)

**Review by dduportal — CHANGES_REQUESTED** [review]
Not needed Kube namespace and Kube secrets

File: publick8s.tf (hunk 1 above)
Comment (smerle33) [reply publick8s.tf]: see commit 16de42e

**Review by dduportal — APPROVED** [review]
`Plan: 7 to add, 0 to change, 0 to destroy.`

LGTM

**Pull request merged**